erpnext/hr/doctype/daily_work_summary/daily_work_summary.py

```diff
@@ -46,13 +46,20 @@ def get_message_details(self):
 		dws_group = frappe.get_doc('Daily Work Summary Group',
 			self.daily_work_summary_group)
 
-		replies = frappe.get_all('Communication',
-			fields=['content', 'text_content', 'sender'],
-			filters=dict(reference_doctype=self.doctype,
-				reference_name=self.name,
-				communication_type='Communication',
-				sent_or_received='Received'),
-			order_by='creation asc')
+		replies = frappe.db.sql("""
```

Comment:
```suggestion
		replies = frappe.db.sql("""
			SELECT `tabcommunication`.content, 
				`tabcommunication`.text_content, 
				`tabcommunication`.sender 
			FROM `tabcommunication` 
				INNER JOIN `tabdynamic link` 
					ON `tabcommunication`.name = `tabdynamic link`.parent 
			WHERE `tabdynamic link`.link_doctype = '%(doctype)s' 
				AND `tabdynamic link`.link_name = '%(name)s' 
				AND `tabcommunication`.communication_type = 'Communication' 
				AND `tabcommunication`.sent_or_received = 'Received' 
			ORDER  BY `tabcommunication`.creation ASC 
		""", {
			"doctype": self.doctype,
			"name": self.name
		}, as_dict=True)
```
@hrwX can we aim such formatting as it makes the code much more readable.

Also, GitHub recognizes this kind of formatting and it gives better highlighting. This makes the review process easier.

Reply:
formatted